cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts

```diff
@@ -154,6 +154,18 @@ describe('Namespace picker', { testIsolation: 'off' }, () => {
     namespacePicker.checkIcon().should('have.length', 1);
   });
 
+  it('can filter after making a selection', () => {
```

Comment:
Looks like you have to add a tag here, oops

Reply:
Indeed, I just the added the following tags that are present in other cases associated with the namespace filter

```
{ tags: ['@explorer2', '@adminUser', '@standardUser'] }
```